Mathlib/AlgebraicGeometry/EllipticCurve/Singular/Cusp.lean

```diff
@@ -0,0 +1,122 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.AlgebraicGeometry.EllipticCurve.Affine.Point
+
+/-!
+
+# Singular cubics with cusps
+
+We develop the basic theory of singular cubics with cusps.
+
+## TODO (Andrew)
+- classify singular cubics with cusps
+
+-/
+
+namespace WeierstrassCurve
+
+variable {R K : Type*} [CommRing R] [Field K] [DecidableEq K]
+
+variable (R) in
+/-- The standard cusp `y² = x³`. -/
+@[simps]
+def standardCusp : WeierstrassCurve R :=
```

Comment:
Note to myself that this is defined in [an old PR](#13847).

Reply:
The link is not working...